mne/io/edf/tests/test_edf.py

```diff
@@ -338,7 +338,8 @@ def test_load_generator(fname, recwarn):
                  [0, 1], id='int list')])
 def test_edf_stim_ch_pick_up(test_input, EXPECTED):
     """Test stim_channel."""
-    TYPE_LUT = {v[0]: k for k, v in _KIND_DICT.items()}
+    # This is fragile for EEG/EEG-CSD, so just omit csd
+    TYPE_LUT = {v[0]: k for k, v in _KIND_DICT.items() if k != 'csd'}
```

Comment:
why do you need to change anything here?

Reply:
having to change something outside seems like a code smell. @larsoner any thought? why is this needed?

Reply:
Kind dict is probably not the best thing to iterate over here. v[0] is EEG_CH here for both eeg and csd types. So in master this maps EEG_CH to 'eeg'. After this PR it will be eeg or csd depending on iteration order unless you exclude csd. And these test files only have eeg so it makes some sense.

It was the minimal change to get it to work but I can make it clearer if it would help